utils/installer/lv-config.example.lua

```diff
@@ -28,6 +28,12 @@ O.plugin.indent_line.active = false
 O.plugin.zen.active = false
 O.plugin.zen.window.height = 0.90
 
+-- nvimtree
+O.nvimtree.side = "left"
```

Comment:
I think it is better to comment these lines out. to show that you can change them
cause that would [change](https://github.com/ChristianChiarulli/LunarVim/blob/rolling/utils/installer/install.sh#L115) the behavior for new installs.

Reply:
Cool! got it. I have made new commits to comment it out.